features/createroom/impl/src/main/kotlin/io/element/android/features/createroom/impl/root/CreateRoomRootView.kt

```diff
@@ -199,6 +213,20 @@ fun CreateRoomActionButtonsList(
     }
 }
 
+@Composable
+fun CreateRoomSearchResultItem(
+    matrixUser: MatrixUser,
+    modifier: Modifier = Modifier,
+    onClick: () -> Unit = {},
+) {
+    MatrixUserRow(
+        modifier = modifier.heightIn(min = 56.dp),
```

Comment:
Should we use a fixed height instead? `modifier.height(56.dp)`

Reply:
By doing this, the text was cropped. I updated the component paddings for a better rendering and do not set the height anymore. Also, I would prefer to not set a fixed height to prevent the same kind of issue, especially if the text size is higher on the user's device.